sympy/parsing/latex/lark/transformer.py

```diff
@@ -97,7 +98,7 @@ def relation(self, tokens):
         elif relation_type == "GTE":
             return sympy.Ge(tokens[0], tokens[2])
         else:
-            raise LaTeXParsingError() # TODO: Fill descriptive error message.
+            raise LaTeXParsingError("An unsupported relational symbol was found.")
```

Comment:
I'm not sure if the rule is really reachable, this already exhausts all tokens, 
so the error should be meaningful for some future

`relation: _expression (EQUAL | NOT_EQUAL | LT | LTE | GT | GTE) _expression`

Reply:
That was what I was going for. Can you give me some suggestions on how to improve the error message?

Reply:
I may suggest in other ways to organize the grammar
from
```lark
relation: _expression (EQUAL | NOT_EQUAL | LT | LTE | GT | GTE) _expression
```
to
```lark
relation: eq | ne | lt | le | gt | ge
eq: _expression EQUAL _expression
ne: _expression EQUAL _expression
...
```

which eliminates the need to write the if-statements inside transformers.

Reply:
That sounds good to me. I'm currently finishing up a PR which adds derivative support, among many other things. I'll make this change in that PR.